zcbor/zcbor.py

```diff
@@ -2692,7 +2703,16 @@ def parse_args():
         "-b", "--default-bit-size", required=False, type=int, default=32, choices=[32, 64],
         help="""Default bit size of integers in code. When integers have no explicit bounds,
 assume they have this bit width. Should follow the bit width of the architecture
-the code will be running on."""
+the code will be running on.""")
+    code_parser.add_argument(
+        "-I", "--include-dir", action="append", type=Path, default=list(), nargs="+",
```

Comment:
I assume `-i` was already taken?

Reply:
I used `-I` because it is very related to the use of `-I` in GCC.